xml/System.Net/WebRequest.xml

```diff
@@ -555,7 +555,7 @@
 ## Remarks  
  The <xref:System.Net.WebRequest.Create%2A> method returns a descendant of the <xref:System.Net.WebRequest> class determined at run time as the closest registered match for `requestUri`.  
   
- For example, when a URI beginning with `http://` or `https://` is passed in `requestUri`, an <xref:System.Net.HttpWebRequest> is returned by <xref:System.Net.WebRequest.Create%2A>. If a URI beginning with `ftp://` is passed instead, the <xref:System.Net.WebRequest.Create%2A> method will return a <xref:System.Net.FileWebRequest> instance. If a URI beginning with `file://` is passed instead, the <xref:System.Net.WebRequest.Create%2A> method will return a <xref:System.Net.FileWebRequest> instance.  
+ For example, when a URI beginning with `http://` or `https://` is passed in `requestUri`, an <xref:System.Net.HttpWebRequest> is returned by <xref:System.Net.WebRequest.Create%2A>. If a URI beginning with `ftp://` is passed instead, the <xref:System.Net.WebRequest.Create%2A> method will return a <xref:System.Net.FtpWebRequest> instance. If a URI beginning with `file://` is passed instead, the <xref:System.Net.WebRequest.Create%2A> method will return a <xref:System.Net.FileWebRequest> instance.  
```

Comment:
wondering if you also wanna change line 570 to say .NET supports instead of The .NET Framework. We're slowly trying to make the content cross-platform and cross-.NET. 😄

Reply:
Yes, in fact there were a lot of such changes, Do you mind taking a look to see whether I broke anything. For example is it ok to say ".NET's" (as a possessive)?

Reply:
good q. we've never used it before. @rpetrusha what do you think?

Reply:
Only in a case where it formerly said "the .NET Framework's", of course.